packages/phpcs-standards/SiteKit/Sniffs/PHPUnit/RequireAssertionMessageSniff.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Class RequireAssertionMessageSniff
+ *
+ * @package Google\Site_Kit\PHPCS\Sniffs\PHPUnit
+ */
+
+namespace Google\Site_Kit\PHPCS\Sniffs\PHPUnit;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+use PHPUnit\Framework\Assert;
+use ReflectionMethod;
+
+/**
+ * Class RequireAssertionMessageSniff
+ *
+ * Ensures that PHPUnit assertions include a message parameter.
+ */
+class RequireAssertionMessageSniff implements Sniff {
+
+	/**
+	 * List of PHPUnit assertion methods.
+	 *
+	 * @var array
+	 */
+	protected $assertion_methods = array();
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		// Get all methods from the Assert class and filter for assertion methods.
+		$this->assertion_methods = array_filter(
+			get_class_methods( Assert::class ),
+			function ( $method ) {
+				return 0 === strpos( $method, 'assert' );
+			}
+		);
```

Comment:
Checking just the `Assert` class is not enough because we also have a custom assert method defined in the `TestCase` class that also has the `$message` argument: https://github.com/google/site-kit-wp/blob/44f18cd3a5261134af55adeda6194d6dfeb12f46/tests/phpunit/includes/TestCase.php#L224-L228 
We need to make sure such custom assert methods are not ignored.